factfinder/main.py

```diff
@@ -378,9 +378,12 @@ def calculate_c_e_m_p_z(self, v: Variable, geotype: str) -> pd.DataFrame:
             df = (
                 self.calculate_e_m(v.pff_variable, geotype)
                 if not (
-                    v.pff_variable in self.special_variables
-                    and geotype in self.aggregated_geography
+                    (v.pff_variable in self.special_variables
+                    and geotype in self.aggregated_geography)
+                    or (v.pff_variable == 'wrkrnothm')
```

Comment:
This works, but for flexibility and legibility maybe we should make another property?

Reply:
I don't that's necessary as of now, we don't have another use case for this